packages/widget-message-meet/Gruntfile.js

```diff
@@ -47,6 +58,8 @@ module.exports = function configGrunt(grunt) {
     });
 
   grunt.registerTask('build', [
+    'clean:dist',
+    'env:build',
```

Comment:
I'm pretty sure this won't impact webpack build because it sets the NODE_ENV after webpack.config's values are set. I had to set NODE_ENV when calling `grunt build` to get around that issue.